modules/stack/src/stack_app.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Prokofeva Elizaveta
+
+#include "include/stack.h"
+#include "include/stack_app.h"
+
+#include <cstring>
+#include <sstream>
+#include <stdexcept>
+#include <string>
```

Comment:
Заголовки нужно поменять местами:

```suggestion
#include "include/stack_app.h"

#include <cstring>
#include <sstream>
#include <stdexcept>
#include <string>

#include "include/stack.h"
```